.devops/code-review-pipelines.yml

```
@@ -89,7 +89,7 @@ stages:
               echo "##[debug] Change placeholder for ecommerce-payment-method-service set branch: $(BRANCH_NAME)"
               sed -i "s/^ECOMMERCE_PAYMENT_METHODS_COMMIT_SHA=.*$/ECOMMERCE_PAYMENT_METHODS_COMMIT_SHA=$(BRANCH_NAME)/" .env
             displayName: 'Change payment-method branch name ecommerce-local .env'
-          - script: docker compose up pagopa-ecommerce-payment-methods-service pagopa-gec-mock pagopa-npg-mock mongo redis -d
+          - script: docker compose up pagopa-ecommerce-payment-methods-service pagopa-gec-mock pagopa-npg-mock mongo redis
```

```suggestion
          - script: docker compose up pagopa-ecommerce-payment-methods-service pagopa-gec-mock pagopa-npg-mock mongo redis -d
```
restore this command

---

Done.